src/ray/gcs/gcs_server/gcs_resource_scheduler.cc

```diff
@@ -128,12 +151,52 @@ absl::flat_hash_set<NodeID> GcsResourceScheduler::FilterCandidateNodes(
   return result;
 }
 
-const std::vector<ResourceRequest> &GcsResourceScheduler::SortRequiredResources(
+std::vector<int> GcsResourceScheduler::SortRequiredResources(
     const std::vector<ResourceRequest> &required_resources) {
-  // TODO(ffbin): A bundle may require special resources, such as GPU. We need to
-  // schedule bundles with special resource requirements first, which will be implemented
-  // in the next pr.
-  return required_resources;
+  std::vector<int> sorted_index(required_resources.size());
+  std::iota(sorted_index.begin(), sorted_index.end(), 0);
+  std::sort(sorted_index.begin(), sorted_index.end(), [&](int b_idx, int a_idx) {
```

Comment:
Can you comment why b_idx comes first here? (comment it is to put things in a reversed order)